frontend/src/components/user-details/components/user-profile/components/avatar-wrapper/avatar-wrapper.tsx

```diff
@@ -58,7 +57,7 @@ const AvatarWrapper: FC = () => {
           btnType="upload"
           onFileSelect={handleFileSelect}
         />
-        <Button btnColor="blue" label="Save" />
+        <Button btnColor="blue" label="Save" onClick={handleAvatarUpdate} />
```

Comment:
```suggestion
        <Button btnColor="blue" label="Change avatar" onClick={handleAvatarUpdate} />
```
On our daily call it was decided to change the label for the button

Reply:
@fshabanov it was done in another ticket